src/app/api/v1/auth/change-password/route.ts

```diff
@@ -0,0 +1,22 @@
+import { changePassword } from "@/services/forgotPasswordCodes";
+import { handleError } from "@/utils/errorHandler";
+import { validateRoutes } from "@/utils/validateRoute";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(req: NextRequest) {
```

Comment:
```suggestion
export async function PATCH(req: NextRequest) {
```

Please update the frontend client code to reflect this!

Reply:
changed in both routeMap and the route handler